Formula/calicoctl.rb

```diff
@@ -0,0 +1,27 @@
+class Calicoctl < Formula
+  desc "Calico CLI tool"
+  homepage "https://www.projectcalico.org"
+  url "https://github.com/projectcalico/calicoctl.git",
+    :tag => "v1.6.1",
+    :revision => "1724e011ac0e608190d7d5512ab8028bcd18ae7b"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    pkg_name = "github.com/projectcalico/calicoctl"
+    build_dir = buildpath/"src/#{pkg_name}"
+    build_dir.install buildpath.children
+    cd build_dir do
+      system "glide", "install", "-strip-vendor"
+      system "make", "binary"
+      bin.install "dist/calicoctl-darwin-amd64" => "calicoctl"
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.

Reply:
I see you've got that text expansion working.

Reply:
What would be the best place to create a feature request to document `prefix.install_metafiles` in the [docs](https://docs.brew.sh/Formula-Cookbook.html)?
Perhaps https://github.com/Homebrew/homebrew.github.io?
Thanks!

Reply:
> I see you've got that text expansion working.

I added it to my GitHub quick replies 🙈.

> What would be the best place to create a feature request to document prefix.install_metafiles in the docs?

It's basically a bug. I've thought about submitting a documentation tweak for it before but that feels like an admission that it's never gonna get fixed.